modules/logging-loki-storage-swift.adoc

```diff
@@ -1,21 +1,20 @@
 // Module is included in the following assemblies:
-// logging/cluster-logging-loki.adoc
+// logging/log_storage/installing-log-storage.adoc
 //
 :_mod-docs-content-type: PROCEDURE
 [id="logging-loki-storage-swift_{context}"]
 = Swift storage
```

Comment:
Procedure titles should begin with a gerund, such as "Configuring Swift storage".

Reply:
Out of scope